src/elements/Svg.tsx

```diff
@@ -81,16 +86,46 @@ export default class Svg extends Shape<SvgProps> {
     root && root.setNativeProps(props);
   };
 
-  toDataURL = (callback: (base64: string) => void, options?: object) => {
+  /**
+   * @deprecated use size property instead.
+   * @description Converts the SVG component to a data URL in either JPEG or PNG format.
+   * @param {function(string): void} callback - A function that receives the base64-encoded data URL string.
+   * @param {DataUrlOptions} options - The options to specify the format (JPEG or PNG) and additional parameters.
```

Comment:
It's a different function. Above JSDoc description is related to deprecated signature:
```ts
toDataURL(callback: (base64: string) => void, options: Size): void;
```